src/components/forms/file-dropper/file-dropper.tsx

```diff
@@ -15,21 +10,28 @@ import { IllustrationType } from '../../ui/illustration/illustration-set';
 import { Trans, useTranslation } from 'gatsby-plugin-react-i18next';
 import { TextStyles } from '../../typography';
 import { FileListItem } from './file-list-item';
-
-interface FileDropperType {
+import {
+  FieldErrors,
+  UseFormClearErrors,
+  UseFormRegister,
+  UseFormSetError,
+  UseFormSetValue,
+} from 'react-hook-form';
+import { Label, StyledErrorMessage } from '../text-input/text-input';
+
+/**
+ * This Interface is needed, since the Props are declared in the new Career-fomr.tsx
+ * I will remove it once the new career form replaces the old one.
+ */
+interface FormDataProps {
+  documents: FileDropperType[];
+}
+export interface FileDropperType {
   file: File;
-  fileCategory: string | null;
+  fileCategory?: { value: string; label: string };
```

Comment:
As this is used at several places, you can put it into it's own type: 

```
interface FileCategory {
  value: string;
  label: string;
}
```

Reply:
Looks better, thanks 👍